src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -37,7 +37,7 @@ export default class SaveInProgressIntro extends React.Component {
           <div>
             <div className="usa-alert usa-alert-info schemaform-sip-alert">
               <div className="usa-alert-body">
-                You can save this form in progress, and come back later to finish filling it out.
+                <strong>Note:</strong> As you are signed in to your account, we can prefill part of your application based on your account details. You can also save your form in progress, and come back later to finish filling it out.
```

Comment:
Since we don't always have prefill information, we should conditionally show this new message. There's a `prefillsAvailable` array in the profile that you can check to make sure the current form is in (the render method for this component does that for another reason already).

Reply:
Thanks @jbalboni, change made.